eoscostarica/eos-rate
# Add ricardian contract

### GH Issue

#472 #471 

- Add Ricardian Contract component (from eoscr-components)
- Add icons in help page

### Steps to test

1. yarn
2. run *make dev*
3. go to help page and see the icons
4. go to terms of use page and see the content

### Checklist
